lib/SimpleSAML/Logger.php

```diff
@@ -453,20 +461,20 @@ private static function createLoggingHandler($handler = null)
      */
     private static function log($level, $string, $statsLog = false)
     {
-        if (self::$loggingHandler === false) {
+        if (self::$initializing === false) {
```

Comment:
And then this should just be `if (self::$initializing)`, as that's what this code is aiming to capture, the case where we're trying to log something while initializing the logging handler.